src/vm/docs/contracts.rs

```diff
@@ -85,20 +85,68 @@ is expired, or if there's never been such a stacker, then returns none."),
+        ("name-transfer", "Changes the name's public key hash. You would send one of these transactions if you wanted to:
+* Change your private key
+* Send the name to someone else
+
+When transferring a name, you have the option to also clear the name's zone file hash (i.e. set it to null). This is useful for when you send the name to someone else, so the recipient's name does not resolve to your zone file."),
+        ("name-revoke", "Makes a name unresolvable. The BNS consensus rules stipulate that once a name is revoked, no one can change its public key hash or its zone file hash.  The name's zone file hash is set to null to prevent it from resolving. You should only do this if your private key is compromised, or if you want to render your name unusable for whatever reason."),
+        ("name-renewal", "Depending in the namespace rules, a name can expire. For example, names in the .id namespace expire after 2 years. You need to send a name reveal every so often to keep your name.
+
+You will pay the registration cost of your name to the namespace's designated burn address when you renew it.
+When a name expires, it enters a month-long \"grace period\" (5000 blocks). 
+
+It will stop resolving in the grace period, and all of the above operations will cease to be honored by the BNS consensus rules.
+You may, however, send a NAME_RENEWAL during this grace period to preserve your name.
+If your name is in a namespace where names do not expire, then you never need to use this transaction."),
+        ("get-namespace-price", "Get the price for a namespace."),
+        ("get-name-price", "Get the price for a name."),
+        ("can-namespace-be-registered", "Returns true if the provided namespace is available."),
+        ("is-name-lease-expired", "Return true if the provided name lease is expired."),
+        ("can-name-be-registered", "Returns true if the provided name can be registered."),
+        ("name-resolve", "Get name registration details."),
+        ("get-namespace-properties", "Get namespace properties."),
+    ];
+
     let pox_skip_display = vec![
         "set-burnchain-parameters",
         "minimal-can-stack-stx",
         "get-reward-set-size",
         "get-reward-set-pox-address",
     ];
 
-    HashMap::from_iter(vec![(
-        "pox",
-        ContractSupportDocs {
-            descriptions: HashMap::from_iter(pox_descriptions.into_iter()),
-            skip_func_display: HashSet::from_iter(pox_skip_display.into_iter()),
-        },
-    )])
+    let bns_skip_display = vec![
+        "namespace-update-function-price",
+        "namespace-revoke-function-price-edition",
+        "check-name-ops-preconditions",
+        "is-name-in-grace-period",
+        "can-receive-name",
```

Comment:
This function should be documented because it helps  to determine whether the current user has (already) registered a name

Reply:
added now